### DISCUSSION TITLE
Would you like to participate in the Learning Equality's open-source community survey?

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#11152] [Community]
Hello Learning Equality friends,

in the next couple of weeks we will be preparing a survey which we're planning to share with Kolibri ecosystem contributors.

We're grateful for all the contributions we are receiving. Cooperation with open-source community is an important part of our vision. In the survey, we'd like to explore various ways of acknowledging contributions in a manner that's supportive and helpful for our contributors. It will also touch on another areas to help us improve our processes and documentation, and you'll be encouraged to give us any feedback on your experience of contributing to Learning Equality products.

I hope it's alright that I'm mentioning some of you who have cooperated with us in the last three months. And if I didn't mention someone who'd like to join, you're very much welcome to do so. Please let us know here if you'd be interested in receiving the survey.

Thank you
Michaela (a software developer at Learning Equality)

@Devanshu-Augusty, @Ghat0tkach, @im-NL, @Jaspreet-singh-1032, @katkuskris, @Shashank245, @GarvitSinghal47, @dbnicholson, @pwithnall, @mhasbini

**Comment by im-NL** [comment]
I would be happy to participate! Happy to be of assistance :)

**Comment by Ghat0tkach** [comment]
Yes @MisRob ,I am grateful for the assistance provided which made my open source contribution possible.! I'll be grateful to participate in the survey

**Comment by dbnicholson** [comment]
Sure, happy to take the survey!

**Comment by GarvitSinghal47** [comment]
Sure, i would be happy to take the survey!

**Comment by Jaspreet-singh-1032** [comment]
Yes, sure.

**Comment by vkWeb** [comment]
@Devanshu-Augusty, @Ghat0tkach, @im-NL, @Jaspreet-singh-1032, @katkuskris, @Shashank245, @GarvitSinghal47, @dbnicholson, @pwithnall, @mhasbini

If you guys have not joined our dev-community slack channel, then please email us to -- ~dev@learningequality.org~ info@learningequality.org and we will add you to our slack channel. Thanks for everything guys!

**Reply by Ghat0tkach** [reply]
Hey @vkWeb  , I have emailed at prescribed address however I have received the following response

```

We're writing to let you know that the group you tried to contact (dev) may not exist, or you may not have permission to post messages to the group. A few more details on why you weren't able to post:

 * You might have spelled or formatted the group name incorrectly.
 * The owner of the group may have removed this group.
 * You may need to join the group before receiving permission to post.
 * This group may not be open to posting.

If you have questions related to this or any other Google Group, visit the Help Center at https://support.google.com/a/learningequality.org/bin/topic.py?topic=25838.

```

**Reply by MisRob** [reply]
Hi @Ghat0tkach, thank you for reporting this, we're looking into it

**Reply by MisRob** [reply]
@Ghat0tkach So the correct e-mail will now be info@learningequality.org

**Comment by pwithnall** [comment]
I am happy to take a survey

**Comment by MisRob** [comment]
Thank you, everyone for volunteering to take the survey, we really appreciate it. We'll let you know as soon as it's ready in the next several weeks.

**Comment by Devanshu-Augusty** [comment]
Yes, i am happy to take the survey

**Comment by MisRob** [comment]
We're finalizing the survey and will be sending it out soon. Thanks everyone!

**Comment by MisRob** [comment]
Here's a new thread with the introduction to the survey and the link to it https://github.com/learningequality/kolibri/discussions/11320. Thank you, everyone!
